src/Installers/Windows/Wix.targets

```
@@ -2,7 +2,7 @@
   <!-- Set versioning properties after Arcade SDK targets have been imported. -->
   <PropertyGroup>
     <!-- Used for generating stable upgrade codes for bundles -->
-    <Version>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).$(AspNetCorePatchVersion).0</Version>
+    <Version>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).$(AspNetCorePatchVersion).$(FileRevisionVersion)</Version>
```

I don't think this is what's used. For 3.1, we should fix this:
```
<GenerateGuid NamespaceGuid="$(NamespaceGuid)" Values="$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion);$(Platform)">
  <Output TaskParameter="Guid" PropertyName="BundleUpgradeCode" />
</GenerateGuid>
```

For example, if we're building the following bundles:

3.1.0.0 x64 should yield GUID A
3.1.20.78567 x64 should yield GUID A
3.1.20.78567 x86 should yield GUID B (x64 is SxS with x86) within the same major.minor range)
3.2.0.0 x64 should yield GUID C (making it SxS with any 3.1 build)

---

Just curious: What sets `$(FileRevisionVersion)`❔

And, a nit if we need this change:
```suggestion
    <Version>$(AspNetCoreMajorMinorVersion).$(AspNetCorePatchVersion).$(FileRevisionVersion)</Version>
```

---

https://github.com/dotnet/aspnetcore/blob/0250c22b9a8d7739ce4e2c75dd3e9d4c74e74f80/src/Installers/Windows/Wix.targets#L7